_studio/shared/src/libmfx_allocator_vaapi.cpp

```diff
@@ -179,7 +179,9 @@ static bool isFourCCSupported(unsigned int va_fourcc)
         case VA_FOURCC_P208:
         case VA_FOURCC_P010:
         case VA_FOURCC_AYUV:
+#if defined (MFX_ENABLE_FOURCC_RGB565)
         case VA_FOURCC_RGB565:
+#endif // MFX_ENABLE_FOURCC_RGB565
```

Comment:
Comment "// MFX_ENABLE_FOURCC_RGB565" is not needed

Reply:
Done